codec_specs.md

```diff
@@ -439,7 +439,9 @@ Description: VP9 is an open and royalty free video compression format developed
 
 Codec BlockAdditions: A single-channel encoding of an alpha channel **MAY** be stored in `BlockAdditions`. The `BlockAddId` of the `BlockMore` containing these data **MUST** be 1.
 
-Initialization: none
+Initialization: The `CodecPrivate` **MUST** contain a list of specific VP9 codec features as described in the "VP9 Codec Feature Metadata" section of [@!WebMContainer].
+Legacy muxers **MAY** not have provided the `CodecPrivate`. In that case it **SHOULD** be reconstructed by the decoder from the first frame.
```

Comment:
```suggestion
In the case `CodecPrivate` is not present, the decoder **SHOULD** reconstruct it from the first frame.
```

MAY should not be used there as it is something a muxer conforming to spec usually may do, so a bit of rewording, similar to what was written in FFV1 for similar issues.

Reply:
```suggestion
Due to the existence of some old non compliant muxers which were not providing `CodecPrivate` , the decoder **SHOULD** reconstruct it from the first frame in the case `CodecPrivate` is not present.
```

Sorry, I forgot the second part of my thoughts.
Not "legacy" because it is like if it was at some point valid.

Reply:
> Otherwise I'll switch it to SHOULD

I don't know enough VP9, but if it is not as e.g. AVC which has SPS at every key frame but if  list of specific VP9 codec features is only in the first frame, I prefer MUST for a muxer because it is not good to have to read the first frame if we want to seek immediately to another frame.

Reply:
> I don't know enough VP9, but if it is not as e.g. AVC which has SPS at every key frame but if list of specific VP9 codec features is only in the first frame

The "Level, Profile, Bit Depth and Chroma Subsampling values" information described in the [WebM Guidelines](https://www.webmproject.org/docs/container/#webm-guidelines) are not supposed to change throughout the whole `Segment`. I don't know about the framing either, but I except at least keyframes to have this information. And on WebM it's a **SHOULD** so it is assuming that starting in the middle of a file without the `CodecPrivate` should still play fine.

Reply:
I applied your last proposal.